test/cql-pytest/test_wasm.py

```diff
@@ -121,162 +122,8 @@ def test_fib(cql, test_keyspace, table1, scylla_with_wasm_only):
 def test_fib_called_on_null(cql, test_keyspace, table1, scylla_with_wasm_only):
     table = table1
     fib_name = unique_name()
-    fib_source = f"""
-(module
```

Comment:
It would be better to store the sources only and compile them to wasm as part of the build.